docker-compose.yaml

```diff
@@ -35,6 +35,6 @@ services:
       context: .
       dockerfile: ./Dockerfile.h2-proxy
     volumes:
-      - ./docker/httpd/httpd.conf:/usr/local/apache2/conf/httpd.conf:ro
+      - ./docker/h2-proxy/default.conf:/etc/nginx/conf.d/default.conf:ro
```

Comment:
As the file seems to be copied during the image building, is it really needed?

Reply:
Yes. Copying is for prod, the volume is for dev (you change the file, reboot the service and it's taken into account without having to rebuild the image)

Reply:
That's my first guess. 😄 
Maybe we can add the same notice as for the nginx service (`# Comment out the next line in production`).